modules/ROOT/nav.adoc

```diff
@@ -1,7 +1,19 @@
 .xref:index.adoc[API Functional Monitoring]
 * xref:index.adoc[About API Functional Monitoring]
+* xref:afm-public-versus-private.adoc[Differences in the Monitoring of the Endpoints of Public and Private APIs]
 * xref:afm-in-anypoint-platform.adoc[API Functional Monitoring in API Monitor]
  ** xref:afm-create-monitor.adoc[Create a Monitor]
+ ** xref:afm-create-private-location.adoc[Create a Private Location]
+ ** xref:afm-add-schedule.adoc[Add a Schedule]
+ ** xref:afm-modify-schedule.adoc[Modify a Schedule]
+ ** xref:afm-delete-schedule.adoc[Delete a Schedule]
+ ** xref:afm-view-schedules-private-location.adoc[View the Schedules That Use a Private Location]
+ ** xref:afm-delete-private-location.adoc[Delete a Private Location]
+ ** xref:afm-delete-test.adoc[Delete a Version of a Test]
+ ** xref:afm-run-test-now.adoc[Run a Test Immediately]
+ ** xref:afm-view-test-history.adoc[View a Test's History]
```

Comment:
This is a single quote, but you should use an apostrophe `’`

Reply:
@runnabro , that is an apostrophe in a monotype font.